proto/zetachain/zetacore/pkg/chains/chains.proto

```diff
@@ -86,6 +92,7 @@ enum Consensus {
   tendermint = 1;
   bitcoin = 2;
   op_stack = 3;
+  solana_consensus = 4;
```

Comment:
This is a bit ugly and inconsistent with the rest. Thing is you can't have a enum value with the same name as one value of another enum in the same file (here solana in network enum) so we might need to do some refactoring to change all values

Reply:
> which also seems weird so probably better to just change all values

IMO this is the best.
We can also change all value without breaking compatibility in the future

Reply:
What I've done in the past and is very intuitive on this case is to just prefix the values with the name of the enum, as in Consensus_solana, Network_solana, etc.

Reply:
Yeah but then in appear as `Consensus_consensus_solana` in the generated code.

Which is not a big problem per se

Reply:
Created an issue for this one: https://github.com/zeta-chain/node/issues/2420